### ISSUE TITLE
DataViews: Frame title should match current view

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#56199]
For instance in this view:

<img width="1694" alt="Screenshot 2023-11-16 at 10 22 04" src="https://github.com/WordPress/gutenberg/assets/846565/b04992d2-749e-4bdd-9428-a79128967b29">

The frame title should be "Drafts" instead of "Pages":

<img width="1694" alt="Screenshot 2023-11-16 at 10 23 09" src="https://github.com/WordPress/gutenberg/assets/846565/4344e461-0194-4a37-8930-5b5e1b70b70e">

**Comment by oandregal** [comment]
I actually have a PR to remove it at https://github.com/WordPress/gutenberg/pull/56241 :) 

Related thread https://github.com/WordPress/gutenberg/pull/56157#issuecomment-1816201054

> I think we could do with actually removing the page title, I've prepared a PR before seeing this one https://github.com/WordPress/gutenberg/pull/56241 The title seems to me a bit redundant with the sidebar info (hierarchy/status: "Pages in draft").

**Comment by jameskoster** [comment]
Reopening. This one is about the title of the white content area, #59011 was about the sidebar titles.

**Comment by jameskoster** [comment]
Another reason to close this one; Pages and Templates data views are currently inconsistent with Patterns. As you navigate through Pattern categories the frame header updates accordingly, unlike Page and Templates.

**Comment by Souptik2001** [comment]
Hi @jameskoster ,

I have raised a PR for this considering the comment you mentioned over here -https://github.com/WordPress/gutenberg/pull/56241#issuecomment-1816762258.

So, can you please review the PR at your convenience.

**Not related to the above PR, but just adding a personal observation over here -**

**Please ignore! I found this issue - https://github.com/WordPress/gutenberg/issues/60468 This one states exactly the same thing. 😅**

As you can see in the below video, when I change the view to suppose Draft and then clear out the Draft filter, still the active view remains as Draft. Because of this the left sidebar and also the subtitle I added remains unchanged.

Thinking about this I think this is what is expected, that's because the views are meant to be kind of preset filters. So, anyone can select a preset filter and then edit the filters, but still ultimately we are in that preset filter only. So, yeah it makes sense. But just wanted to put this here.

https://github.com/WordPress/gutenberg/assets/39427067/960138bf-6fcf-4ce9-a3ea-937928e1de40

Thanks!

**Comment by youknowriad** [comment]
@jameskoster is this issue still valid, maybe we should close it with our desire to move to tabs.

**Comment by jameskoster** [comment]
Yes, if we move to tabs this can be closed. Is there an issue for that?

**Comment by youknowriad** [comment]
@jameskoster I don't know, didn't you create an issue exploring the layout or something, maybe a dedicated issue would be best.

**Comment by jameskoster** [comment]
Oh yeah. I made a new issue here: https://github.com/WordPress/gutenberg/issues/73251.

Closing this in favor of that.
